benchmarks/nhcal_acceptance/config.yml

```diff
@@ -0,0 +1,50 @@
+sim:nhcal_acceptance:
+  extends: .det_benchmark
+  stage: simulate
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+        INDEX: ["00","01","02","03","04","05","06","07","08","09"]
+  script:
+    - snakemake --cores 5 sim_output/nhcal_acceptance/E${ENERGY}/sim_epic_backward_hcal_only.${INDEX}.edm4hep.root
+
+bench:nhcal_acceptance_combine:
+  extends: .det_benchmark
+  stage: benchmarks
+  needs: 
+    - "sim:nhcal_acceptance"
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+  script:
+    - snakemake --cores 5 sim_output/nhcal_acceptance/sim_epic_backward_hcal_only_E${ENERGY}_combined_10files.edm4hep.root
+
+
+bench:nhcal_acceptance_analysis:
+  extends: .det_benchmark
+  stage: benchmarks
+  needs:
+    - "bench:nhcal_acceptance_combine"
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+  script:
+    - snakemake --cores 3 results/nhcal_acceptance/analysis_epic_backward_hcal_only_E${ENERGY}_combined_10files.pdf
+
+collect_results:nhcal_acceptance:
+  extends: .det_benchmark
+  stage: collect
+  needs: 
+    - "bench:nhcal_acceptance_analysis"
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+  script:
+    - ls -lrht
+    - mv results{,_save}/ # move results directory out of the way to preserve it
+    - snakemake $SNAKEMAKE_FLAGS --cores 1 --delete-all-output results/nhcal_acceptance/analysis_epic_backward_hcal_only_E${ENERGY}_combined_10files.pdf
+    - mv results{_save,}/
+    - |
+      gs -sDEVICE=pngalpha -dUseCropBox -r144 \
+        -o 'results/nhcal_acceptance/analysis_epic_backward_hcal_only_E${ENERGY}_combined_10files.png' \
```

Comment:
```suggestion
        -o "results/nhcal_acceptance/analysis_epic_backward_hcal_only_E${ENERGY}_combined_10files.png" \
```
Bash variable expansion doesn't work in single quotes. This results in a bug:
<img width="589" alt="image" src="https://github.com/user-attachments/assets/a953c580-0ac9-49ad-8d81-ad9b733bb4c8" />